classes/book.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'item'
+
+class Book < Item
+  def initialize(publisher, cover_state)
+    super(published_date, archived)
```

Comment:
@skabeo , the `published_date` and `archived` properties don't yet exist in this context.
I know you only forgot to pass them into the `Book` constructor, before sending them to `super()` 😄

Reply:
Thank you. I'm on it